test/functional/ui/inccommand_spec.lua

```diff
@@ -2634,3 +2634,38 @@ it(':substitute with inccommand, timer-induced :redraw #9777', function()
     :%s/foo/ZZZ^                   |
   ]])
 end)
+
+it('long :%s/ with inccomand does not collapse cmdline', function()
+  local screen = Screen.new(10,5)
+  clear()
+  common_setup(screen)
+  command('set inccommand=nosplit')
+  feed(':%s/AAAAAAA')
+  feed('A')
+  feed('A')
+  feed('A')
+  feed('A')
+  feed('A')
+  feed('A')
+  feed('A')
+  feed('A')
+  feed('A')
+  feed('A')
+  feed('A')
+  feed('A')
+  feed('A')
+  feed('A')
+  feed('A')
+  feed('A')
+  feed('A')
+  feed('A')
+  feed('A')
+  feed('A')
```

Comment:
`feed()` actually takes multiple args and will call them as separate RPC requests. So this could be:

    feed('A', 'A', 'A', 'A', 'A', 'A', 'A', ...)